README.md

```diff
@@ -130,6 +130,100 @@ or for all users:
 
   `sogo-restore-user /var/lib/sogo/backups/sogo-2017-12-10_0030/ -A`
 
+## Customizing theme and logo
+
+The files for the theme `custom_theme.js` and the logo `logo-full.svg` to customize the SOGo instance `sogo1` need to be located in `/home/sogo1/.config/state` and owned by user `sogo1`.
+
+Here is an example of a `custom_theme.js`:
+
+```
+(function() {
+    'use strict';
+
+    angular.module('SOGo.Common')
+      .config(configure)
+
+    /**
+     * @ngInject
+     */
+    configure.$inject = ['$mdThemingProvider'];
+    function configure($mdThemingProvider) {
+      var blueMap = $mdThemingProvider.extendPalette('indigo', {
+        '200': '#3e4044',
+        '300': '#292b30',
+        '1000': '#4C566A',
+        '50': '#313131',
+        'A100': '#444444',
+        '500': '#00558c',
+        'contrastDefaultColor': 'light',
+        'contrastLightColors': '50 100 200 300 400 500 600 A100 A200 A400 A700',
+        'contrastStrongLightColors': '700 800 900 A700'
+      });
+      $mdThemingProvider.definePalette('frost-grey', blueMap);
+
+      $mdThemingProvider.theme('default')
+        .primaryPalette('indigo', {
+          'default': '400',
+          'hue-1': '400',
+          'hue-2': '600',
+          'hue-3': 'A700'
+        })
+        .accentPalette('grey', {
+          'default': '400',
+          'hue-1': '300',
+          'hue-2': '500',
+          'hue-3': 'A700'
+        })
+        .backgroundPalette('grey');
+
+      $mdThemingProvider.generateThemesOnDemand(false);
+    }
+  })();
+```
+
+The sogo-app service file needs to be edited to add the files as volumes:
+
+    runagent -m sogo1 systemctl --user edit sogo-app
```

Comment:
--full is better understandable I think

`runagent -m sogo1 systemctl --user edit --full sogo-app`

Reply:
Yes but then it's not persistent to survive updates as the service file will be overwritten.

Reply:
├── apache.service.d
│   └── override.conf
you are right, it does appear only when I use --edit WTF :p